examples/platform/linux/BUILD.gn

```diff
@@ -14,11 +14,11 @@
 
 import("//build_overrides/chip.gni")
 
-config("chip_examples_project_linux_platform_main_config") {
+config("chip_app_main_config") {
   include_dirs = [ "." ]
```

Comment:
Why say `chip` ? Everything in this project is related to CHIP.

Note: Here is where you can say

```
import("${chip_root}/examples/common/pigweed/pigweed_rpcs.gni")

if (chip_enable_pw_rpc) {
  defines = [ "PW_RPC_ENABLED" ]
}
```

Totally fine to share that, as long as the `defines` is set here rather than on the `executable`.

Reply:
I'm a bit concerned about that what if someone added `chip_enable_pw_rpc=true` when building apps other than the lightingapp.
I guess other apps also need the pw rpc.

Reply:
Either way seems ok.